### PR TITLE
add nats-init component

### PR DESCRIPTION
**Pull request by k15r**
**Description**

Changes proposed in this pull request:

- nats-init was part of event bus, with event-bus gone we need to put it in its own component
**Related issue(s)**

#7730

### CODE REVIEW

File: components/nats-init/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.11
+LABEL source=git@github.com:kyma-project/kyma.git
+
+WORKDIR /nats-streaming/
+
+COPY scripts/prepare-config.sh .
```
Comment (sayanh) [inline components/nats-init/Dockerfile:6]: Why don't we need the licences?
```
COPY licenses ./licenses
```

Comment (k15r) [reply components/nats-init/Dockerfile:6]: which licenses would you copy in?

Comment (sayanh) [reply components/nats-init/Dockerfile:6]: The licences which are pulled [here](https://github.com/kyma-project/kyma/blob/master/common/makefiles/generic-make-go.mk#L84).

Comment (k15r) [reply components/nats-init/Dockerfile:6]: `docker.mk` does not pull any licenses. This target is only valid in `generic-make-go.mk`

Comment (sayanh) [reply components/nats-init/Dockerfile:6]: The question is don't we need the licenses anymore? Cause we had licenses in the previous image of nats-init.

Comment (sayanh) [reply components/nats-init/Dockerfile:6]: Please feel free to ignore my comments if needed. I don't wanna block this PR. It is just not clear to me why we don't need the licenses anymore.

Comment (k15r) [reply components/nats-init/Dockerfile:6]: it was empty in the previous images as well

Comment (k15r) [reply components/nats-init/Dockerfile:6]: so, i checked our license puller script https://github.com/kyma-project/test-infra/blob/master/prow/scripts/license-puller.sh
there is just no code that would pull a license for a bash only image.

File: components/nats-init/Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+APP_NAME = nats-init
+APP_PATH = components/$(APP_NAME)
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190930-d28d219
```
Comment (nachtmaar) [inline components/nats-init/Makefile:3]: (nit) If we only use `docker.mk`, is `BUILDPACK` and `APP_PATH` still required? From my understanding only `APP_NAME` and `SCRIPTS_DIR` is required.

Comment (nachtmaar) [inline components/nats-init/Makefile:3]:
```suggestion
```

**Pull request merged**